patches/react-native-share-menu+6.0.0.patch

```diff
@@ -25,12 +77,31 @@ index 74badda..6ed5073 100644
              print("Error: \(NO_APP_GROUP_ERROR)")
              return
          }
-@@ -102,7 +102,7 @@ class ShareMenu: RCTEventEmitter {
+@@ -100,10 +100,12 @@ class ShareMenu: RCTEventEmitter {
+ 
+     @objc(getSharedText:)
      func getSharedText(callback: RCTResponseSenderBlock) {
-         var data = [DATA_KEY: sharedData] as [String: Any]
-
+-        var data = [DATA_KEY: sharedData] as [String: Any]
++        var data = [DATA_KEY: sharedData as Any] as [String: Any]
+         
 -        if let bundleId = Bundle.main.bundleIdentifier, let userDefaults = UserDefaults(suiteName: "group.\(bundleId)") {
-+        if let appGroup = Bundle.main.object(forInfoDictionaryKey: HOST_APP_GROUP_INFO_PLIST_KEY), let userDefaults = UserDefaults(suiteName: (Bundle.main.object(forInfoDictionaryKey: appGroup as! String) as! String)) {
-             data[EXTRA_DATA_KEY] = userDefaults.object(forKey: USER_DEFAULTS_EXTRA_DATA_KEY) as? [String: Any]
+-            data[EXTRA_DATA_KEY] = userDefaults.object(forKey: USER_DEFAULTS_EXTRA_DATA_KEY) as? [String: Any]
++        if let appGroup = Bundle.main.object(forInfoDictionaryKey: HOST_APP_GROUP_INFO_PLIST_KEY) as? String {
++            if let userDefaults = UserDefaults(suiteName: appGroup) {
++                data[EXTRA_DATA_KEY] = userDefaults.object(forKey: USER_DEFAULTS_EXTRA_DATA_KEY) as? [String: Any]
++            }
          } else {
              print("Error: \(NO_APP_GROUP_ERROR)")
+         }
+diff --git a/node_modules/react-native-share-menu/ios/Modules/ShareMenuReactView.swift b/node_modules/react-native-share-menu/ios/Modules/ShareMenuReactView.swift
+index e290cce..59f2974 100644
+--- a/node_modules/react-native-share-menu/ios/Modules/ShareMenuReactView.swift
++++ b/node_modules/react-native-share-menu/ios/Modules/ShareMenuReactView.swift
+@@ -45,6 +45,7 @@ public class ShareMenuReactView: NSObject {
+         }
+ 
+         extensionContext.completeRequest(returningItems: [], completionHandler: nil)
++        ShareMenuReactView.detachViewDelegate()
```

Comment:
It looks like this patch is missing a few lines from the original:

1. More calls to `detachViewDelegate`, which Ryan added to avoid some crashes: https://github.com/infinitered/ExpensifyApp/pull/63/files#diff-1aa046d161a1081bd541198ac073c301a3773ce2f3591f5f88a5a6fc85c02e36R150. Not all of these might be necessary now.
2. file type handling: https://github.com/infinitered/ExpensifyApp/pull/63/files#diff-1aa046d161a1081bd541198ac073c301a3773ce2f3591f5f88a5a6fc85c02e36R168

@trevor-coleman Let me know if you have bandwidth to update this, if not, I'll do it tomorrow. Either way it might be good/less confusing to break this up into multiple patches since patch-package can do that now, which is exciting.